driver/layernorm_driver.hpp

```diff
@@ -371,19 +438,14 @@ int LayerNormDriver<Tgpu, Tref>::RunBackwardGPU()
 template <typename Tgpu, typename Tref>
 Tref LayerNormDriver<Tgpu, Tref>::GetTolerance()
 {
-    if(data_type == miopenHalf)
-    {
-        return 1e-3;
-    }
-    else if(data_type == miopenFloat)
-    {
-        return 5e-5;
-    }
-    else if(data_type == miopenBFloat16)
-    {
-        return 5e-3;
-    }
-    return 0;
+    // Computation error of fp16 is ~2^13 (=8192) bigger than
+    // the one of fp32 because mantissa is shorter by 13 bits.
+    auto tolerance = (sizeof(Tgpu) == 4 || sizeof(Tgpu) == 1) ? 1.5e-6 : 8.2e-3;
```

Comment:
Have you meant 2?
```C++
    ... || sizeof(Tgpu) == 2 ...
```
`sizeof(Tgpu) == 1` maps to FP8 type.

Reply:
> `sizeof(Tgpu) == 1` maps to FP8 type.

Correction: it maps to FP8, BF8 and INT8 (consequences are trivial I guess ;)

Reply:
fixed it.